manim/mobject/three_dimensions.py

```diff
@@ -134,6 +134,30 @@ def set_fill_by_checkerboard(self, *colors, opacity=None):
 
 
 class Sphere(ParametricSurface):
+    """A sphere
+
+    Examples
+    ---------
+    .. manim:: ExampleSphere
+        :save_last_frame:
+
+        class ExampleSphere(ThreeDScene):
+            def construct(self):
+                self.set_camera_orientation(phi = PI/6, theta = PI/6)
+                sphere1 = Sphere(center = (3,0,0),radius = 1,resolution = (30, 40),u_min = 0.001,u_max = PI - 0.001,v_min = 0,v_max = TAU)
+                sphere1.set_color(RED)
+                self.add(sphere1)
+                sphere2 = Sphere(center=(0,1,0),radius=2,resolution=(40, 50))
```

Comment:
What's the reasoning behind specifying `center` and then using `move_to` ? (same goes for sphere3)

Reply:
Removed the move_to functions.